build_scripts/build_opencv.sh

```diff
@@ -29,18 +29,22 @@ cmake -DCMAKE_BUILD_TYPE=RELEASE \
       -DWITH_IPP=OFF \
       -DWITH_OPENCL=OFF \
       -DWITH_GTK=OFF \
-      -DBUILD_JPEG=OFF \
+      -DBUILD_JPEG=ON \
```

Comment:
Isn't it on by default? I don't think we need or should retype default settings - especially ones that are very unlikely to change (like enabling JPEG support).